.github/workflows/release.yaml

```diff
@@ -74,11 +74,6 @@ jobs:
         with:
           target: x86_64
           args: --release --locked --out dist
-      - name: "Test wheel - x86_64"
-        run: |
-          pip install dist/${{ env.PACKAGE_NAME }}-*.whl --force-reinstall
-          ruff --help
-          python -m ruff --help
```

Comment:
Unfortunately we can no longer run the x64_64 wheel (since this is an ARM runner, we're cross-compiling), but I think it's fine and worth it (we already run this test in the Universal branch below).

Reply:
IMO if the M1 runner has Rostta2 enabled it should run fine? At least for `ruff --help`.

Reply:
Lemme look back at what failed...

Reply:
I think it was the `pip install dist/${{ env.PACKAGE_NAME }}-*.whl --force-reinstall`, saying that the wheel isn't supported for the current platform.

Reply:
Oh right, the python interpreter it installed might not be universal2, if it is you can probably use `arch -x86_64 python` to switch to x86_64 mode.

Reply:
I appreciate you chiming in, you're always so helpful.